easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-foss-2018b.eb

```diff
@@ -0,0 +1,54 @@
+name = 'HEALPix'
+version = '3.50'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['Healpix_3.50_2018Dec10.tar.gz']
+patches = ['HEALPix_noXtest.patch']
+checksums = [
+    'ec9378888ef8365f9a83fa82e3ef3b4e411ed6a63aca33b74a6917c05334bf4f',  # Healpix_3.50_2018Dec10.tar.gz
+    'ec82c8b2beb80937c83d2e545e26ade08b3647ab5a3401c8703e6523b0782c08',  # HEALPix_noXtest.patch
+]
+
+dependencies = [('CFITSIO', '3.45')]
+
+sanity_check_paths = {
+    'files': [
+        'bin/alteralm',
+        'bin/anafast',
+        'bin/hotspot',
+        'bin/map2gif',
+        'bin/median_filter',
+        'bin/plmgen',
+        'bin/sky_ng_sim',
+        'bin/sky_ng_sim_bin',
+        'bin/smoothing',
+        'bin/synfast',
+        'bin/ud_grade',
+        'lib/libchealpix.a',
+        'lib/libgif.a',
+        'lib/libhealpix.a',
+        'lib/libhpxgif.a',
+        'lib/libchealpix.so',
+    ],
+    'dirs': ['include'],
+}
+
+sanity_check_commands = [
+    ('cd %(installdir)s && make test &> build/testresults.txt'),
+    ('[ "$(cat %(installdir)s/build/testresults.txt | grep -c \'test completed\')" -eq "4" ]'),
+    ('[ "$(cat %(installdir)s/build/testresults.txt | grep -c \'normal completion\')" -eq "10" ]')
+]
+
+modextrapaths = {
+    'PATH': 'src/cxx/generic_gcc/bin',
+    'LIBRARY_PATH': 'src/cxx/generic_gcc/lib',
+    'CPATH': 'src/cxx/generic_gcc/include',
```

Comment:
This looks like more stuff for the easyblock, since it's critical to making things work correctly, and applicable to many versions going forward?

Reply:
then I should do some testing with older versions.

Actually, if you suspect there is demand for high quality easyblock for HEALPIX, there is more work to do here - E.G. to support more variants of build (generic_gcc|optimized_gcc).

Reply:
moved to easyblock in https://github.com/easybuilders/easybuild-easyblocks/pull/1763